lib/commands/new.js

```diff
@@ -79,6 +79,7 @@ module.exports = Command.extend({
       analytics: this.analytics,
       tasks: this.tasks,
       project: Project.nullProject(this.ui, this.cli),
+      ciProvider: 'travis',
```

Comment:
This shouldn't be defaulted here, it should be defaulted up in `availableOptions`